src/pks/flow/richards_pk.cc

```diff
@@ -86,6 +86,7 @@ Richards::Richards(Teuchos::ParameterList& pk_tree,
   sat_gas_key_ = Keys::readKey(*plist_, domain_, "saturation gas", "saturation_gas");
   sat_ice_key_ = Keys::readKey(*plist_, domain_, "saturation ice", "saturation_ice");
 
+  
```

Comment:
Can you just revert richards_pk.cc to what it is on master, e.g.

    git checkout origin/master -- src/pks/flow/richards_pk.cc

No need for any changes to this PK.

Reply:
fixed